account_invoice_refund_link/README.rst

```diff
@@ -0,0 +1,92 @@
+=================================
+Link refund invoice with original
+=================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--invoicing-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-invoicing/tree/13.0/account_invoice_refund_link
+    :alt: OCA/account-invoicing
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-invoicing-13-0/account-invoicing-13-0-account_invoice_refund_link
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/95/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module shows the links between refunds and their original invoices in the
+invoice form and also keep track of refund lines and their original invoice
+lines.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+This module creates in the invoice form a new 'Refunds' page when the invoice
+has some refunds. It shows a line for each refund invoice with main
+information.
+
+For refunds, it shows the refund reason and original invoice link.
+
```

Comment:
Mention also that refund lines are linked with originator invoice lines for easy reporting and other purposes.

Reply:
Now, that links are not visualized for users, should I mention that anyway?